### DISCUSSION TITLE
Jukebox mode:  Best Feature Ever

### DISCUSSION THREAD

**Discussion by john-** [jcorporation/myMPD#65] [General]
Thanks for the jukebox feature!    For the last 20+ years (literally) I have implemented jukebox capability in the audio servers I have worked on.   The first audio server I wrote used lpd to manage the queue and printer commands to work the playlist!   Even with that I coded a jukebox function.  When I moved to mpd there were a few programs I wrote to add random songs to the queue.

With the recent myMPD It looks like that history of rolling my own jukebox is now over!

Thanks!

I don't know of a place to say thanks so will put my thanks in as a feature request and then close it.

**Comment by jcorporation** [comment]
Many thanks, such a feedback is realy motivating!
